addons/payment_mollie/const.py

```diff
@@ -66,4 +66,5 @@
     'bank_transfer': 'banktransfer',
     'p24': 'przelewy24',
     'sepa_direct_debit': 'directdebit',
+    'twint': 'twint',
```

Comment:
Do we really need to add this mapping if pm code and code from mollie side is same? :thinking:

Reply:
Oh yes, it's working without this too. Thanks for pointing it out :+1: